Update 2 (#249)
NilPointer-Software/bdfd-wiki: src/bdscript/addTimestamp.md

# $addTimestamp
Adds a timestamp to an embed.

## Syntax
```
$addTimestamp[(index)]
```

## Parameter

> - `index`: What embed the timestamp should belong to. _(Optional)_ [(learn more)](../resources/embedIndexes.md)

Reviewer: ```suggestion
- `index`: To which embed the timestamp should be added to. _(Optional)_ [(learn more)](../resources/embedIndexes.md)
```
